cfg/config.go

```diff
@@ -84,19 +86,21 @@ var (
 // NewDefaultConfig retrieves the config with sane defaults
 func NewDefaultConfig() *Configuration {
 	return &Configuration{
-		MountPath:               "/data",
-		BackupAnnotation:        "k8up.syn.tools/backup",
-		BackupCommandAnnotation: "k8up.syn.tools/backupcommand",
-		FileExtensionAnnotation: "k8up.syn.tools/file-extension",
-		ServiceAccount:          "pod-executor",
-		BackupCheckSchedule:     "0 0 * * 0",
-		GlobalKeepJobs:          6,
-		BackupImage:             "quay.io/vshn/wrestic:latest",
-		PodExecRoleName:         "pod-executor",
-		RestartPolicy:           "OnFailure",
-		MetricsBindAddress:      ":8080",
-		PodFilter:               "backupPod=true",
-		EnableLeaderElection:    true,
+		MountPath:                        "/data",
+		BackupAnnotation:                 "k8up.syn.tools/backup",
+		BackupCommandAnnotation:          "k8up.syn.tools/backupcommand",
+		FileExtensionAnnotation:          "k8up.syn.tools/file-extension",
+		ServiceAccount:                   "pod-executor",
+		BackupCheckSchedule:              "0 0 * * 0",
+		GlobalKeepJobs:                   6,
```

Comment:
Why still use the deprecated value? My expectation was to have `GlobalKeepJobs` unset and have the others set instead.